domains/iarasuster.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "iarasuster",
+        "email": "iarasuster@gmail.com"
+    },
+    "record": {
+        "URL": "iarasuster.is-a.dev"
```

Comment:
Invalid URL record (missing protocol)
also that would cause a redirect loop

Reply:
what should I change it to?

Reply:
the URL of the website you want the domain to redirect to